### PR TITLE
Budget app 

### PR DESCRIPTION
**Pull request by dennis-every**
In this pull request, we completed the following activities for the Microverse budget-app project:

- Set up the repository and tools.
- Create your models according to the ERD diagram. 
- Add validations and tests.
- Implement the authentication and authorization.
- Create the page to add a category.
- Create the home page.
- Create the page to add an expense.
- Create the expenses page.
- Make sure tests are still passing.
- Deploy the project
- Record a video of the project.
- Add README and a pull request for review.

### CODE REVIEW

File: app/views/categories/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  <main>
+    <ul>
+      <% @categories.each do |category| %>
+        <%= link_to category_path(category) do %>
+          <li class='card'>
+            <div class="card--icon">
+              <i class="fa-solid fa-house"></i>
+            </div>
+            <div class='card--description'>
+              <p><b><%= category&.name %></b></p>
```
Comment (thecodechaser) [inline app/views/categories/index.html.erb]: - [ ] On the home page(categories page) I can see the house icon for all the categories but it should display the icon uploaded by the user for that category. Please make sure to fix this issue. (You're displaying a static icon for all categories on this code, You should use the link uploaded by the user here).

image

File: app/views/expenses/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<section class='categories'>
+  <header>
+    <%= link_to sanitize("&larr;"), @category, class: 'icon' %>
+    <h1>New Expense</h1>
+    <span></span>
+  </header>
+  <main>
+    <%= render 'form', expense: @expense %>
+  </main>
```
Comment (thecodechaser) [inline app/views/expenses/new.html.erb:9]: - [ ] Please fix the below issues for the Transactions page.
 - Please make sure to display the amount of all transactions with transaction details.
 - Please make sure the total amount for the category is displayed at the top as requested in the project requirements, Currently, it's at the bottom.
image


### Your design

image

### Given design

image

File: README.md [hunk 3]
```diff
@@ -1,24 +1,156 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for the information provided to build this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (thecodechaser) [inline README.md]: - [ ] Please make sure to give appropriate credit to the author of the original design idea to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) as requested in the project requirements.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @dennis-every👋, 

### You are doing a marvelous job on the RoR capstone project and this is one of the best projects I have seen so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- Highly organized code. ✔️
- Gitflow is followed. :heavy_check_mark:
- Nice UI design. ✔️
- Professional README file. :heavy_check_mark:
- No linter errors. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️

- [x] The maximum length of the presentation video should be 5 minutes as requested in the project requirements but the length of your presentation video is 6:30+ minutes. Please make sure the maximum length of the video is 5 minutes.

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi @dennis-every 😊, 
# STATUS: APPROVED! 🥇 💯 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**